tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/test_deploy_ovf.yml

```diff
@@ -1,26 +1,27 @@
 ---
 - name: Get VM Info
-  community.vmware.vmware_guest_info:
+  vmware.vmware.guest_info:
     hostname: "{{ vcenter_hostname }}"
     username: "{{ vcenter_username }}"
     password: "{{ vcenter_password }}"
-    datacenter: "{{ deploy_ovf_datacenter_name }}"
     name: "{{ item }}"
     validate_certs: false
   register: guest_info
+  retries: 3
+  delay: 10
 
 - name: Check Properties Of VM {{ item }}
   ansible.builtin.assert:
     that:
-      - guest_info.instance.hw_cluster == deploy_ovf_cluster_name
-      - guest_info.instance.hw_folder == deploy_ovf_folder
-      - guest_info.instance.hw_name == item
-      - guest_info.instance.hw_datastores == [deploy_ovf_datastore]
-      - guest_info.instance.hw_memtotal_mb == test_vm_hardware.memory_mb
-      - guest_info.instance.hw_processor_count == test_vm_hardware.num_cpus
+      - guest_info.guests[0].hw_cluster == deploy_ovf_cluster_name
```

Comment:
I know we talked about it in the past, but it still feels that we broke the API here - if the return is a list and not one element...

Reply:
the return values are not backwards compatible with the community modules. the original guest_info always returned a list of vms that matched the search criteria, so it is backwards compatible with that

Reply:
I'm talking about compatible with the community.... 
@machacekondra what do you think?

Reply:
Well, I think we don't have to be backward compatible and I think it's not the target. We want to refactor the code so it's better then the community (like separate `vmware_guest`), so if it's better to have different API, let's go for it. Maybe for some small changes it's nice to document the difference, if user want to migrate.